adjacent_material.md

```diff
@@ -231,3 +232,27 @@ Challenge](https://www.youtube.com/watch?v=Souzjv6WfrY)
 
 [Building a ML Text to Image App with Stable Diffusion in 15 Minutes \|
 Coding Challenge](https://www.youtube.com/watch?v=7xc0Fs3fpCg)
+
+## Conclusion
+
+The fast.ai course is a great introduction to the world of AI and ML. It
+is a dense course, covering lots of concepts and models, which requires
+some re-watching and additional material to fully grasp - some of which
+we presented here.
+
+When we started the study group, my knowledge of AI and ML was very limited.
+Of course, as a developer, I had heard about - and used - chatbots (ChatGPT,
+Bard); I had used GitHub Copilot and found it amazing at "guessing" the next
```

Comment:
This is my personal experience, hence the use of I instead of 'we'. But I can change that if you want.

Reply:
Makes senses. I'd leave it.